modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/item/FileEntryContentDashboardItem.java

```diff
@@ -296,6 +299,8 @@ public String getScopeName(Locale locale) {
 	@Override
 	public Map<String, Object> getSpecificInformation(Locale locale) {
 		return HashMapBuilder.<String, Object>put(
+			"clipboard", _getClipboard()
```

Comment:
Hey @beltranrengifo !!

We should change it, we should add the webDAV URL as a field and the new latest version URL, we should erase every mention to clipboard.

Reply:
You should change the frontend as we mention in the daily meeting, so we can read URLs in the frontend side.

And delete the latest URL that now is being shown in the info panel.

Once you finish the frontend changes, please, start with the versions, so we can do as much frontend changes as possible before your PTO next week.

Reply:
The frontend is what I need from you, because we don't have any frontend in the next week.

In the backend side, we have plenty of hands.

thanks!!

Reply:
Yeah, you're right, thought it was faster that it is. Sorry for the confusion